src/libs/ReportUtils.js

```diff
@@ -1341,17 +1341,34 @@ function getChatRoomSubtitle(report) {
  * @param {Object} report
  * @returns {String}
  */
-function getParentNavigationSubtitle(report) {
+function hasParentNavigationSubtitle(report) {
```

Comment:
`hasParentNavigationSubtitle` and `getParentNavigationSubtitleData` are pretty much the same. It's more efficient to just relay on `getParentNavigationSubtitleData` then having to call both functions.